server/routes/nl/helpers.py

```diff
@@ -47,17 +47,34 @@
 from server.lib.nl.detection.utils import create_utterance
 import server.lib.nl.fulfillment.fulfiller as fulfillment
 import server.lib.nl.fulfillment.utils as futils
+from server.lib.shared import names
 from server.lib.translator import detect_lang_and_translate
 from server.lib.translator import translate_page_config
 from server.lib.util import get_nl_disaster_config
 from server.routes.nl import helpers
+from server.routes.shared_api.place import get_place_type
 import server.services.bigtable as bt
 from shared.lib.constants import EN_LANG_CODE
 import shared.lib.utils as shared_utils
 
 _SANITY_TEST = 'sanity'
 
 
+# Get the default place to be used for fulfillment. If there is a place in the
+# request, use that. Otherwise, use pre-chosen places.
+def _get_default_place(request: Dict, is_sdg: bool):
+  default_place_dcid = request.args.get('default_place', default='', type=str)
+  if default_place_dcid:
+    place_type = get_place_type(default_place_dcid)
+    place_name = names([default_place_dcid])[default_place_dcid]
```

Comment:
Sorry for missing this before, instead of 2 API calls, I think we can just call place.get_place_from_dcids() which will use `get_place_info` to return `Place` objects.

Reply:
oh, that's a useful method, updated!